chiptopher/guet
# Rotate pairs

## Overview
Enable the ability to rotate between which pair is set as the commit author, outlined in #3 .

### Notes
Closes #3 

## Testing Instructions
Set multiple committers, commit multiple times, and then observe the commit logs to see that the author changes.
